### DISCUSSION TITLE
Schneider Wiser OTA updates

### DISCUSSION THREAD

**Discussion by dennisgranasen** [Koenkk/zigbee2mqtt#16312] [Devices]
Hi,

Has anyone investigated ota updates for Schneider Wiser devices. AFAIK the only known way to update these devices is via the hub. Perhaps sniffing the traffic could reveal the firmware urls?

**Comment by PSTB** [comment]
I do not know if it has already been done, but I guess you might need a hub for a chance to see this. I see my Schneider Wiser devices ask for OTA but there's definitively no URL to be found there.

**Reply by dennisgranasen** [reply]
Yes, I assume that the hub is requesting updates from some url. Without it there won't be anything to sniff. Sadly I don't own one.

**Comment by idoo** [comment]
Hey, found that article https://fearthecow.net/projects/clipsal-wiser-updating-without-hub-zigbee-home-bridge-home-assistant/
maybe we could use the same approach
